File: IOS/SAIOT/SAIOT/DataModel/RemoteControllerInfo.swift
//
//  RemotecontrollerInfo.swift
//  SAIOT
//
//  Created by 양창엽 on 2018. 4. 2..
//  Copyright © 2018년 Yang-Chang-Yeop. All rights reserved.
//

import RealmSwift

class RemoteControllerInfo: Object {
    
    // MARK: - Integer Value
    @objc dynamic var channelType: Int = 0
    
    // MARK: - String Value
    @objc dynamic var name: String = ""
    @objc dynamic var remoteType: String = ""
    
    // MARK: - Date Value
    @objc dynamic var createTime: Date = Date()
}

File: Android/SAIOT/app/src/main/java/androidtown/org/saiot/RemoteControllerInfo.java
package androidtown.org.saiot;
import java.util.Date;
import io.realm.RealmObject;

/**
 * Created by Ei Seok on 2018-04-24.
 */

public class RemoteControllerInfo extends RealmObject {
    //Integer Value
    private int channelType;




    //String Value

    private String name;




    //Date Value
    private Date createTime=new Date();


}
